qa/tasks/cephfs/filesystem.py

```diff
@@ -378,8 +378,8 @@ def mds_signal(self, mds_id, sig, silent=False):
     def mds_is_running(self, mds_id):
         return self.mds_daemons[mds_id].running()
 
-    def newfs(self, name='cephfs', create=True):
-        return Filesystem(self._ctx, name=name, create=create)
+    def newfs(self, name='cephfs', create=True, **kwargs):
+        return Filesystem(self._ctx, name=name, create=create, **kwargs)
```

Comment:
It will be nice to add a comment about what arguments are expected through `kwargs`.

Reply:
added